contracts/curve/AnteCurveSTETHRugTest.sol

```diff
@@ -0,0 +1,23 @@
+pragma solidity ^0.8.0;
+
+import "../AnteTest.sol";
+
+/// @title Curve stETH Doesn't Rug
+contract AnteSTETHCurveRugTest is AnteTest("Curve stETH Keeps 99% of it's ETH.") {
+    // https://etherscan.io/address/0xde0b295669a9fd93d5f28d9ec85e40f4cb697bae
+    address public stETHCurveSwap = 0xDC24316b9AE028F1497c275EB9192a3Ea0f67022;
+
+    // 2022-04-05: stETH Curve Contract has 800k ETH, so -99% is ~8k ETH
+    uint256 public constant RUG_THRESHOLD = 8 * 1000 * 1e18;
```

Comment:
While this is good as a rug threshold right now, why not have this set at 1% on deployment, that way we get to stick with the Ante Test title of keeping 99% of it's ETH (from deployment)